packages/mobile-app/data/facades/wallet/oreowalletHandlers.ts

```diff
@@ -0,0 +1,382 @@
+        Network.MAINNET,
+        txnHash,
+      );
+
+      if (!txn) {
+        return null;
+      }
+
+      return {
+        fee: txn.fee,
+        timestamp: new Date(txn.timestamp),
+        expiration: null,
+        hash: txn.hash,
+        block:
+          txn.blockSequence > 0
+            ? {
+                hash: "",
+                sequence: txn.blockSequence,
+              }
+            : null,
+        submittedSequence: 0,
+        assetBalanceDeltas: txn.assetBalanceDeltas.map((abd) => ({
+          assetId: abd.assetId,
+          delta: abd.delta,
+        })),
+        status: txn.status as TransactionStatus,
+        notes: [
+          {
+            sender: txn.sender,
+            owner: txn.receiver,
+            memo: txn.memo ?? "",
+            value: txn.value,
+            spent: false,
+            assetId: "",
+            hash: "",
+            memoHex: "",
+          },
+        ],
+        burns: [],
+        mints: [],
+        spends: [],
+        type: txn.type as TransactionType,
+      };
+    },
+  ),
+  getTransactions: f.handler.query(
+    async ({
+      accountName,
+    }: {
+      accountName: string;
+      options?: {
+        limit?: number;
+        offset?: number;
+        ascending?: boolean;
+        assetId?: string;
+        address?: string;
+      };
+    }): Promise<Transaction[]> => {
+      const txns = await oreoWallet.getTransactions(
+        accountName,
+        Network.MAINNET,
+      );
+
+      return txns.map((txn) => ({
+        fee: txn.fee,
+        timestamp: new Date(txn.timestamp),
+        expiration: null,
+        block:
+          txn.blockSequence > 0
+            ? {
+                hash: "",
+                sequence: txn.blockSequence,
+              }
+            : null,
+        hash: txn.hash,
+        submittedSequence: 0,
+        assetBalanceDeltas: [],
+        status: txn.status as TransactionStatus,
+        notes: [],
+        burns: [],
+        mints: [],
+        spends: [],
+        type: txn.type as TransactionType,
+      }));
+    },
+  ),
+  getWalletStatus: f.handler.query(async (): Promise<WalletStatus> => {
+    const response = await OreowalletServerApi.getLatestBlock(Network.MAINNET);
+    return {
+      status: oreoWallet.scanState.type,
+      latestKnownBlock: Number(response.currentBlockIdentifier.index),
+    };
+  }),
+  importAccount: f.handler.mutation(
+    async ({
+      account,
+      name,
+    }: {
+      account: string;
+      name?: string;
+    }): Promise<Account> => {
+      const importedAccount = await oreoWallet.importAccount(
+        Network.MAINNET,
+        account,
+        name,
+      );
+
+      return {
+        name: importedAccount.name,
+        viewOnly: importedAccount.viewOnly,
+        publicAddress: importedAccount.publicAddress,
+        balances: {
+          iron: {
+            assetId:
+              "51f33a2f14f92735e562dc658a5639279ddca3d5079a6d1242b2a588a9cbf44c",
```

Comment:
suggestion: Could use the new constant here

Reply:
Updated to use it everywhere